purchases/src/main/kotlin/com/revenuecat/purchases/common/networking/ETagManager.kt

```diff
@@ -55,11 +55,11 @@ internal class ETagManager(
 ) {
 
     internal fun getETagHeaders(
-        path: String,
+        urlString: String,
```

Comment:
Although this is only a parameter name change in `EtagManager`, the idea here is to stop using only the path as the key for caching the responses and use the full URL of the request. The reason is that we might be making requests with the same path to different endpoints (e.g. `vi/product_entitlement_mapping`  to both the main server and to the fallback server) and we don't want to return the response of a server for requests to the other. Please, lmk if you don't agree with this change @RevenueCat/coresdk 

Note that the same is done already in iOS (see https://github.com/RevenueCat/purchases-ios/blob/303f7dfb7b27fe24db61a3cfa851e35571447355/Sources/Networking/HTTPClient/ETagManager.swift#L132-L134)

Reply:
Yep. That is a side effect of this. I checked and the cache seems to only be cleaned up whenever the user ID changes. 
In addition, with the current change it would mean that the cache will still (always) contain the previous values.

Do you think it's worth keeping the old cache keys and only add new keys for the endpoints of the new fallback URL? I don't really know what's best

Reply:
Yes, makes sense. Thank you!